pkg/cmd/admin/policy/review.go

```diff
@@ -0,0 +1,195 @@
+package policy
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	"k8s.io/kubernetes/pkg/apis/apps"
+	"k8s.io/kubernetes/pkg/kubectl"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/runtime"
+	utilerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	ometa "github.com/openshift/origin/pkg/api/meta"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/cli/describe"
+	"github.com/openshift/origin/pkg/cmd/templates"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	securityapi "github.com/openshift/origin/pkg/security/api"
+)
+
+var (
+	reviewLong = templates.LongDesc(`Checks which Service Account can create a Pod.
+	The Pod is inferred from the PodTemplateSpec in the provided resource.
+	If no Service Account is provided the one specified in podTemplateSpec.spec.serviceAccountName is used,
+	unless it is empty, in which case "default" is used.
+	If Service Accounts are provided the podTemplateSpec.spec.serviceAccountName is ignored.
+	`)
+	reviewExamples = templates.Examples(`# Check whether Service Accounts sa1 and sa2 can admit a Pod with TemplatePodSpec specified in my_resource.yaml
+	# Service Account specified in myresource.yaml file is ignored
+	$ %[1]s -s sa1,sa2 -f my_resource.yaml
+	
+	# Check whether Service Account specified in my_resource_with_sa.yaml can admit the Pod
+	$ %[1]s -f my_resource_with_sa.yaml
+	
+	# Check whether default Service Account can admit the Pod, default is taken since no Service Account is defined in myresource_with_no_sa.yaml
+	$  %[1]s -f myresource_with_no_sa.yaml
+	`)
+)
+
+const ReviewRecommendedName = "scc-review"
+
+type sccReviewOptions struct {
+	client              client.PodSecurityPolicyReviewsNamespacer
+	namespace           string
+	enforceNamespace    bool
+	out                 io.Writer
+	mapper              meta.RESTMapper
+	typer               runtime.ObjectTyper
+	RESTClientFactory   func(mapping *meta.RESTMapping) (resource.RESTClient, error)
+	printer             kubectl.ResourcePrinter
+	FilenameOptions     resource.FilenameOptions
+	ServiceAccountNames []string
+}
+
+func NewCmdSccReview(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	o := &sccReviewOptions{}
+	cmd := &cobra.Command{
+		Use:     name,
+		Short:   "Checks which ServiceAccount can create a Pod",
+		Long:    reviewLong,
+		Example: fmt.Sprintf(reviewExamples, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(o.Complete(f, args, cmd, out))
+			kcmdutil.CheckErr(o.Run(args))
+		},
+	}
+
+	cmd.Flags().StringSliceVarP(&o.ServiceAccountNames, "serviceaccounts", "s", o.ServiceAccountNames, "List of ServiceAccount names, comma separated")
```

Comment:
Shorthand should be `-z` to be consistent with other commands where serviceaccounts are specified.